packages/wallet/frontend/src/lib/api/grants.ts

```diff
@@ -48,10 +48,21 @@ type GetGrantResult = SuccessResponse<Grant>
 type GetGrantResponse = GetGrantResult | ErrorResponse
 
 type DeleteGrantResponse = SuccessResponse | ErrorResponse
+type ChoiceGrantRequestResponse = SuccessResponse | ErrorResponse
 
 interface GrantsService {
   list: (cookies?: string) => Promise<ListGrantsResponse>
   get: (grantId: string, cookies?: string) => Promise<GetGrantResponse>
+  getInteraction: (
+    interactionId: string,
+    nonce: string,
+    cookies?: string
+  ) => Promise<GetGrantResponse>
+  answerGrantRequest: (
+    interactionId: string,
+    nonce: string,
+    isAccepted: boolean
```

Comment:
I think we can pass directly `accept` or `reject` instead of a boolean and rename the parameter to `action`.